dev_tools/pypath

```diff
@@ -21,6 +21,7 @@
 
 
 if [ -n "${ZSH_VERSION}" ]; then
+    # shellcheck disable=2296,2298
```

Comment:
Why is this line disabled?

Reply:
The comment makes shellcheck skip this if-branch, in effect line 25 -
https://github.com/quantumlib/Cirq/blob/6e8f539ba4461747e738ae534e4bb144a08f6bde/dev_tools/pypath#L23-L25
The nested `${...}` expression there is valid in zsh, but is invalid bash syntax.
shellcheck checks bash syntax only.